data/input_files/default.lua

```diff
@@ -14,12 +14,15 @@ main_mesh = {
 nonlinear_solid = {
     stiffness_solver = {
         linear = {
-            rel_tol     = 1.0e-6,
-            abs_tol     = 1.0e-8,
-            max_iter    = 5000,
-            print_level = 0,
-            solver_type = "gmres",
-            prec_type   = "HypreAMG",
+            type = "iterative",
+            iterative_params = {
```

Comment:
Depending which PR goes in first, these `params` should become `options`. The easier answer is to probably have this one go in first and then I'll fix it.